version/version.go

```diff
@@ -4,4 +4,4 @@ const Maj = "0"
 const Min = "10"
 const Fix = "0"
 
-const Version = "0.10.0"
+const Version = Maj + "." + Min + "." + Fix
```

Comment:
https://github.com/tendermint/tendermint/blob/6312eb91bee05171d97cecd509be115b2c6b6477/scripts/dist.sh#L6

we should probably add a comment here